pkg/utils/panics/panics.go

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Open Component Model contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package panics
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/open-component-model/ocm/pkg/logging"
+)
+
+var ReallyCrash = true
+
+type PanicHandler func(interface{})
+
+var PanicHandlers = []PanicHandler{logHandler}
+
+func HandlePanic(additionalHandlers ...PanicHandler) {
```

Comment:
Here we should also provide an flavor able to forward the panic as an error return (see above)

Reply:
Just be aware though that in that case, we'll have to wrap the returning value. Because the Handler needs to be called in a defer.